packages/amplify-category-auth/src/provider-utils/awscloudformation/auth-stack-builder/auth-stack-transform.ts

```diff
@@ -95,10 +95,7 @@ export class AmplifyAuthTransform extends AmplifyCategoryTransform {
   public applyOverride = async (): Promise<void> => {
     const backendDir = pathManager.getBackendDirPath();
     const overrideDir = path.join(backendDir, this._category, this.resourceName);
-    const isBuild = await buildOverrideDir(backendDir, overrideDir).catch(error => {
-      printer.error(`Build error : ${error.message}`);
-      throw new Error(error);
-    });
+    const isBuild = await buildOverrideDir(backendDir, overrideDir);
```

Comment:
For clarity: buildOverrideDir detects override files, and compiles them if they exist

The compiled code then gets executed later to apply the customer's override logic